config/api.yml

```diff
@@ -394,6 +394,7 @@
     - :collection
     :verbs: *gpppd
     :klass: MiqGroup
+    :parsed_by_attrs: description
```

Comment:
It's not immediately clear to me what this means in this context...maybe a higher level name for this key? (Parsing ties it contextually to some implementation detail - is that important?)